cutplace/applications.py

```diff
@@ -218,4 +238,4 @@ def main_for_script():  # pragma: no cover
     sys.exit(main())
 
 if __name__ == '__main__':
-    main_for_script()
+    main_for_script()
```

Comment:
Check you editor settings, it seems to remove the final newline.

Also run `tox -e flake8` before pushing, it should discover a few issues with this code.